rosbag2_tests/test/rosbag2_tests/test_rosbag2_info_end_to_end.cpp

```diff
@@ -68,6 +68,56 @@ TEST_P(InfoEndToEndTestFixture, info_end_to_end_test) {
       "Serialization Format: cdr"));
 }
 
+TEST_P(InfoEndToEndTestFixture, info_with_verbose_option_end_to_end_test) {
+  internal::CaptureStdout();
+  auto exit_code = execute_and_wait_until_completion(
+    "ros2 bag info bag_with_topics_and_service_events --verbose",
+    bags_path_);
```

Comment:
how about the `warning` case with `--topic --verbose`, it would be better to check if `verbose` prevails to print the service information?